src/urllib3/util/timeout.py

```diff
@@ -95,22 +98,29 @@ class Timeout:
     """
 
     #: A sentinel object representing the default timeout value
-    DEFAULT_TIMEOUT = _GLOBAL_DEFAULT_TIMEOUT
-
-    def __init__(self, total=None, connect=_Default, read=_Default):
+    DEFAULT_TIMEOUT: object = _GLOBAL_DEFAULT_TIMEOUT
```

Comment:
Make this `: _TYPE_TIMEOUT` so it's settable by users via `urllib3.util.Timeout.DEFAULT_TIMEOUT = ...`